ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -223,7 +225,13 @@ extension MSFListView {
         }
 
         var body: some View {
-            Button(action: cell.onTapAction ?? {}, label: {
+            Button(action: cell.onTapAction ?? {
```

Comment:
I think the functionality of having a cell collapse or expand its subcells is great. But what about the header?
I was thinking the header could be a button with a different style (that looks like a header) and the section can be configured as collapsible?

What did you have in mind for those scenarios?

Reply:
I was thinking anything related to the Header/Footer would be worked on as a separate component entirely (treated similarly to current arch). I envision there being different types of Headers/Footers, including an expand/collapse type. I think that could be treated as a separate issue, and still coexist with individual expand/collapse cells. Thoughts?

Reply:
Yes! As we agreed offline, the Header/Footer provide the functionality of sticking at the top while the contents are being scrolled, so we can keep it as a separate issue.
The cells being collapsed and expanded look great!
Is there a GitHub issue for the Header expand/collapse functionality?

Reply:
https://github.com/microsoft/fluentui-apple/issues/402